netbox/extras/models/customfields.py

```diff
@@ -653,7 +656,7 @@ def validate(self, value):
                 if value not in [c[0] for c in self.choices]:
                     raise ValidationError(
                         _("Invalid choice ({value}). Available choices are: {choices}").format(
-                            value=value, choices=', '.join(self.choices)
+                            value=value, choices=', '.join([c[0] for c in self.choices])
```

Comment:
We don't want to return _all_ possible choices here, which could number in the thousands. (This was a problem in the original code.) Instead, we should return a slice with just the first few options to convey a pattern to the user, and annotate the total number of choices.

Reply:
I think I captured this decently enough